src/NuGet.Clients/NuGet.PackageManagement.UI/Models/PackageDetailControlModel.cs

```diff
@@ -317,7 +317,7 @@ public bool IsSelectedVersionInstalled
             {
                 return SelectedVersion != null
                     && InstalledVersion != null
-                    && SelectedVersion?.Range?.OriginalString == InstalledVersionRange?.OriginalString
+                    && (IsProjectPackageReference ? SelectedVersion?.Range?.OriginalString == InstalledVersionRange?.OriginalString : true)
```

Comment:
My understanding of the fix is it's a bypass of the OriginalString comparison only for packages.config.